migrations/20210206_drop_badges_column_add_sharing_flags.go

```diff
@@ -37,6 +37,12 @@ func init() {
 				return err
 			}
 
+			if cfg.Db.Dialect == config.SQLDialectSqlite {
+				// see 20201106_migration_cascade_constraints
+				logbuch.Info("not attempting to drop and regenerate constraints on sqlite")
```

Comment:
Thanks for the contribution! Please adapt the error message to this specific migration as it's not quite accurate here. Afterwards I'm going to merge this.